packages/dropdowns/src/elements/Menu/Menu.tsx

```diff
@@ -108,13 +109,15 @@ const Menu: React.FunctionComponent<IMenuProps & ThemeProps<DefaultTheme>> = pro
     return () => clearTimeout(timeout);
   }, [isOpen, isAnimated]);
 
+  const themeContext = useContext(ThemeContext);
+
   // Reset Downshift refs on every render
   itemIndexRef.current = 0;
   nextItemsHashRef.current = {};
   previousIndexRef.current = undefined;
   itemSearchRegistry.current = [];
 
-  const popperPlacement = isRtl(props)
+  const popperPlacement = themeContext.rtl
```

Comment:
Do you remember if there are any more instances of `isRtl` we should modernize? We could track the rest for a future date if there are any left.

Reply:
I _think_ we got em' all 😅. My "element" component typedef work will tell us for certain. And we'll follow-on deprecate `isRtl` for removal in v9.